js/datepicker.js

```diff
@@ -881,6 +882,8 @@
 			var date = new Date(this.year, this.month, curDay, 0, 0, 0, 0);
 			var longdate = this.formatDate(date, this.options.titleFormat);
 			var curDayClass = curDay == this.date && this.month == this.curMonth && this.year == this.curYear ? ' curDay' : '';
+			today.setHours(0,0,0,0);
```

Comment:
I suppose techncially this doesn't need to be called each time round the loop

Reply:
Yep, good shout. I've moved it.